docs/source/osds/scenarios.rst

```diff
@@ -1,12 +1,232 @@
+
+   osd_scenario: lvm
+   devices:
+     - /dev/sda
+     - /dev/sdb
+     - /dev/nvme0n1
+
+Similar to the initial example, this would end up producing 2 OSDs, but data
+would be placed on the slower spinning drives (``/dev/sda``, and ``/dev/sdb``)
+and journals would be placed on the faster solid state device ``/dev/nvme0n1``.
+
+Other (optional) supported settings:
+
+- ``crush_device_class``: Sets the crush class for all OSDs created with this
+  method (it is not possible to have a per-OSD crush class using the *simple*
+  configuration approach). Values *must be* a string, like
+  ``crush_device_class: "ssd"``
+
+
+Advanced configuration
+^^^^^^^^^^^^^^^^^^^^^^
+
+This configuration is useful when more granular control is wanted when setting
+up devices and how they should be arranged to provision an OSD. It requires an
+existing setup of volume groups and logical volumes (the tooling will **not**
+create these).
+
+To use this configuration, the ``lvm_volumes`` option must be populated with
+logical volumes, volume groups, and optionally partitions for journals, or
+``block.db`` and ``block.wal``.
+
+.. note:: This configuration uses ``ceph-volume lvm create`` to provision OSDs
+
+Supported ``lvm_volumes`` configuration settings:
+
+- ``data``: The logical volume name or full path to a raw device (an LV will be
+  created using 100% of the raw device)
+
+- ``data_vg``: The volume group name. If using a logical volume name for ``data``
+    it is **required**.
+
+- ``crush_device_class``: Device class name for the resulting OSD, allows to
+  individually set the class for each OSD, unlike the global
+  ``crush_device_class`` that sets them for all OSDs.
+
+.. note:: If you wish to change set the ``crush_device_class`` for the OSDs
+          when using ``devices`` you must set it using the global ``crush_device_class``
+          option as shown above. There is no way to define a specific crush device class
+          per OSD when using ``devices`` like there is for ``lvm_volumes``.
+
+
+``filestore`` objectstore only values:
+
+- ``journal``: The logical volume name or full path to a partition.
+
+- ``journal_vg``: The volume group name. If using a logical volume name for
+  ``journal`` it is **required**.
+
+
+``bluestore`` objectstore only values:
+
+- ``db``: The logical volume name or full path to a partition.
+
+- ``db_vg``: The volume group name. If using a logical volume name for ``db``
+    it is **required**.
+
+- ``wal``: The logical volume name or full path to a partition.
+
+- ``wal_vg``: The volume group name. If using a logical volume name for ``wal``
+    it is **required**.
+
+
+.. note:: These ``bluestore`` settings are optional, and are useful if placing
+          them in faster devices. It is possible to create a bluestore OSD with
+          a single raw device.
+
+.. warning:: It is not allowed to use the same values for many OSDs (for
+             example, a journal LV cannot be used for more than one OSD)
```

Comment:
1) I'm not sure how this renders, but it seems odd to be talking about journals bracketed by bluestore.  Do you mean specifically db and wal can't be shared...whereas:
2) can you share a device for filestore journals?  I thought so?

Reply:
Some people thought you can use 1 journal for more than 1 OSD. It was a warning from the previous version of the doc that I wanted to keep around. I'll move it out to a better context